python/postprocessing/examples/example_postproc.py

```diff
@@ -4,7 +4,21 @@
 ROOT.PyConfig.IgnoreCommandLineOptions = True
 from importlib import import_module
 from PhysicsTools.NanoAODTools.postprocessing.framework.postprocessor import PostProcessor
+##soon to be deprecated
+from PhysicsTools.NanoAODTools.postprocessing.modules.jme.jetmetUncertainties import *
+##new way of using jme uncertainty
+from PhysicsTools.NanoAODTools.postprocessing.modules.jme.jetmetHelperRun2 import *
 
 from  exampleModule import *
-p=PostProcessor(".",["../../../../NanoAOD/test/lzma.root"],"Jet_pt>150","keep_and_drop.txt",[exampleModule()],provenance=True)
+
+##Function parameters
+##(isMC=True, dataYear=2016, runPeriod="B", jesUncert="Total", redojec=False, jetType = "AK4PFchs", noGroom=False)
+##All other parameters will be set in the helper module
+
+jmeCorrections = createJMECorrector(True, "2016", "B", "Total", True, "AK4PFchs", False)
+
+fnames=["/eos/cms/store/mc/RunIISummer16NanoAODv5/DYJetsToLL_M-50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8/NANOAODSIM/PUMoriond17_Nano1June2019_102X_mcRun2_asymptotic_v7_ext2-v1/120000/FF69DF6E-2494-F543-95BF-F919B911CD23.root"]
+
+#p=PostProcessor(".",fnames,"Jet_pt>150","keep_and_drop.txt",[jetmetUncertainties2016(),exampleModuleConstr()],provenance=True)
+p=PostProcessor(".",fnames,"Jet_pt>150","keep_and_drop.txt",[jmeCorrections(),exampleModuleConstr()],provenance=True)
```

Comment:
The example crashes because keep_and_drop.txt doesn't include the run, luminosityBlock, and event branches required by jetSmearer.setSeed.  Please remove this parameter from the PostProcessor call or modify keep_and_drop.txt.